aries-site/src/themes/aries.js

```diff
@@ -3,6 +3,40 @@ import { deepMerge } from 'grommet/utils';
 
 export const aries = deepMerge(hpe, {
   defaultMode: 'dark',
+  dataTable: {
+    pinned: {
+      header: {
+        extend: ({ theme }) => `
+        backdrop-filter: blur(8px);
+        @-moz-document url-prefix() {
+        background: ${
+          theme.dark ? 'rgba(64, 75, 92, 0.95)' : 'rgba(255, 255, 255, 0.95)'
+        };
+        }
```

Comment:
Can you add a comment for why this is needed and also provide a link to MDN docs / can-i-use docs for support of `backdrop-filter`? Sounds like FireFox should be supporting shortly - currently an opt in users can enable, but not yet a default.

Reply:
yes I read that Firefox had `backdrop-filter` on its list of priority

Reply:
I will change these values on the PR for the theme and add the comment there then we can strip this out